### DISCUSSION TITLE
Can CORTX run on ARM machine

### DISCUSSION THREAD

**Discussion by proteuszj** [Seagate/cortx#662] [General] [answered]
as title, thanks

**Comment by andriytk** [comment, marked as answer]
Not yet, but we are currently porting Motr to ARM - yes. It's in progress...

**Reply by johnbent** [reply]
@sainarasim and @andriytk and @nezzzu , can we update this please?

**Reply by andriytk** [reply]
The core components of CORTX (like Motr, Hare and RGW) should be able to run on ARM64 by now - yes. But this is an experimental platform. So feel free to try and open tickets if something is not working.
